webapp/src/components/backstage/playbook_edit/automation/toggle.tsx

```diff
@@ -6,6 +6,7 @@ import React from 'react';
 import styled from 'styled-components';
 
 interface ToggleProps {
+    inputId?: string;
```

Comment:
```suggestion
    inputId?: string;
    children: React.ReactNode;
```

(with `children` right after `RoundSwitch`)

Reply:
`Label` would also need additional styling something akin to: 

```css
display: flex;
align-items: center;
column-gap: 12px;
font-weight: inherit;
```